File: Android/Java/UD2/Recursos/app/src/main/java/com/pmdm/recursos/MainActivity.java
package com.pmdm.recursos;

import android.os.Bundle;
import android.widget.ImageView;
import android.widget.LinearLayout;
import android.widget.TextView;
import androidx.appcompat.app.AppCompatActivity;
import androidx.core.content.ContextCompat;
import androidx.core.content.res.ResourcesCompat;
import android.graphics.Typeface;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);

        setContentView(R.layout.activity_main);

        LinearLayout layout2 = findViewById(R.id.Linear2);
        TextView textView2 = findViewById(R.id.Text2);
        ImageView imageView2 = findViewById(R.id.Image2);

        //Cambiar color de fondo
        //layout2.setBackgroundColor(Color.parseColor("#5E7EF1"));
        layout2.setBackgroundColor(ContextCompat.getColor(this,R.color.custom_color));

        //Cambiar fuente
        Typeface typeface = ResourcesCompat.getFont(this,R.font.rubikwetpaint);
        textView2.setTypeface(typeface);

        //Cambiar la imagen
        imageView2.setImageResource(R.drawable.saturno);
    }
}

File: Android/Kotlin/UD2/RecursosKotlin/app/src/main/java/com/pmdm/recursoskotlin/MainActivity.kt
package com.pmdm.recursoskotlin

import android.os.Bundle
import android.widget.ImageView
import android.widget.LinearLayout
import android.widget.TextView
import androidx.appcompat.app.AppCompatActivity
import androidx.core.content.ContextCompat
import androidx.core.content.res.ResourcesCompat

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)

        setContentView(R.layout.activity_main)

        val layout2 : LinearLayout = findViewById<LinearLayout>(R.id.Linear2)
        val textView2 : TextView = findViewById<TextView>(R.id.Text2)
        val imageView2 : ImageView = findViewById<ImageView>(R.id.Image2)

        layout2.setBackgroundColor(ContextCompat.getColor(this,R.color.custom_color))

        val typeface = ResourcesCompat.getFont(this,R.font.rubikwetpaint)
        textView2.setTypeface(typeface)

        imageView2.setImageResource(R.drawable.saturno)
    }
}